src/Xamarin.Android.Build.Tasks/Utilities/AndroidResource.cs

```diff
@@ -20,10 +20,13 @@ public static bool UpdateXmlResource (string res, string filename, Dictionary<st
 				UpdateXmlResource (res, doc.Root, acwMap, additionalDirectories, logMessage, (e) => {
 					registerCustomView?.Invoke (e, filename);
 				});
-				using (var stream = File.OpenWrite (tmpfile))
-					using (var xw = new LinePreservedXmlWriter (new StreamWriter (stream)))
-						xw.WriteNode (doc.CreateNavigator (), false);
-
+				using (var stream = File.OpenWrite (tmpfile)) {
+					using (var sw = new StreamWriter (stream)) {
+						using (var xw = new LinePreservedXmlWriter (sw)) {
+							xw.WriteNode (doc.CreateNavigator (), false);
+						}
+					}
+				}
```

Comment:
This is a nitpick, but sometimes I stack multiple `using` in a row like this:
```
using (var stream = File.OpenWrite (tmpfile))
using (var sw = new StreamWriter (stream))
using (var xw = new LinePreservedXmlWriter (sw)) {
		xw.WriteNode (doc.CreateNavigator (), false);
}
```

Reply:
I'm beginning to like the stack `using` as well.